lib/computer_player.rb

```diff
@@ -43,47 +46,11 @@ def switch(current_marker, opponent_player)
     end
 
     def best_move(current_marker, scores)
-      if current_marker == "X"
+      if current_marker == MARKERS[0]
```

Comment:
Will your computer's marker always be in the first position in the `MARKERS` array? It may be better to assign each marker to its own constant value to make things order independent.